frontend/src/scene/main-view/CourseModal.js

```diff
@@ -156,7 +199,20 @@ const MainModal = ({ course, seletectedDate, onConfirm, clear }) => (
                             {Number(course.price).toLocaleString('fi')} €
                         </span>
                     </div>
-                    <Button alternative bold onClick={onConfirm}>
+
+                    <ErrorMessageTag
+                        key="3"
+                        color={errorDetail.colorCode}
+                        style={{ marginTop: 10 }}
```

Comment:
you can move this to `ErrorMessageTag` definition

Reply:
Moving color will make the color of error detail disappear, other attributes are moved